src/gpodder/util.py

```diff
@@ -73,14 +73,16 @@
 try:
     import html5lib
 except ImportError:
-    logger.warn('html5lib not found, falling back to HTMLParser')
+    error_message = "html5lib was not found, fall-back to HTMLParser"
+    logger.warning(f'{error_message}', stack_info=True)
```

Comment:
The separate var for `error_message` is not needed, just use a string constant argument in the call and break lines if they are longer than 79 chars.

The minimum Python version for gPodder is 3.5, where f-strings are not supported. Please use other formatting methods.

Also, I'm not sure if `stack_info` is really needed here.